mobile/src/bundles/common/components/input/input.tsx

```diff
@@ -5,19 +5,23 @@ import {
     type FieldPath,
     type FieldValues,
 } from 'react-hook-form';
+import { type TextInputProps } from 'react-native';
 import { TextInput } from 'react-native';
 
 import { Text, View } from '~/bundles/common/components/components';
 import { useFormController } from '~/bundles/common/hooks/hooks';
 
 import { styles } from './styles';
 
-type Properties<T extends FieldValues> = {
+type InputType = 'text' | 'number' | 'email' | 'password';
+
+type Properties<T extends FieldValues> = TextInputProps & {
     control: Control<T, null>;
     errors: FieldErrors<T>;
     label: string;
     name: FieldPath<T>;
     placeholder: string;
+    type: InputType;
 };
```

Comment:
```suggestion
type Properties<T extends FieldValues> = TextInputProps & {
    control: Control<T, null>;
    errors: FieldErrors<T>;
    label: string;
    name: FieldPath<T>;
};
```
Let's leave just the keyboard type prop from TextInputProps, placeholder is also already in TextInputProps